src/App.tsx

```diff
@@ -31,65 +31,135 @@ export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  switch (sortType) {
+    case SortType.ALPHABET:
+      visibleGoods.sort();
+      break;
+    case SortType.LENGTH:
+      visibleGoods.sort((a, b) => a.length - b.length);
+      break;
+    default:
+      break;
+  }
+
+  if (isReversed) {
+    visibleGoods.reverse();
+  }
 
   return visibleGoods;
 }
 
 // DON'T save goods to the state
-// type State = {
-//   isReversed: boolean,
-//   sortType: SortType,
-// };
-
-export const App: React.FC = () => {
-  return (
-    <div className="section content">
-      <div className="buttons">
-        <button
-          type="button"
-          className="button is-info is-light"
-        >
-          Sort alphabetically
-        </button>
-
-        <button
-          type="button"
-          className="button is-success is-light"
-        >
-          Sort by length
-        </button>
-
-        <button
-          type="button"
-          className="button is-warning is-light"
-        >
-          Reverse
-        </button>
-
-        <button
-          type="button"
-          className="button is-danger is-light"
-        >
-          Reset
-        </button>
-      </div>
+type State = {
+  isReversed: boolean,
+  sortType: SortType,
+};
+
+// eslint-disable-next-line react/prefer-stateless-function
+export default class App extends React.Component {
```

Comment:
```suggestion
export class App extends React.Component<{}, State> {
```

Reply:
@agniya-i 
![image](https://user-images.githubusercontent.com/101051619/224811487-322204ec-eff8-4217-a5fd-367e3a9c9de3.png)
initially, i used just !state, but it wasn't working for me
that's why I used !this.state

Reply:
> 

![image](https://user-images.githubusercontent.com/101051619/224811725-07747533-99a2-4d71-8c2c-ff6b37a650c0.png)
doesn't work this way